lib/diagnostic.js

```diff
@@ -13,14 +26,36 @@ const min = function min() {
 // and returns the sum of that set
 // assume that all the argument are numbers
 const sum = function sum() {
+  let numHolder = null;
+  if (arguments.length === 0) {
+    numHolder = undefined;
+  } else {
+    numHolder = arguments[0];
+  }
+
+  for (let i = 1; i < arguments.length; i++) {
+    numHolder += arguments[i];
+  }
 
+  return numHolder;
```

Comment:
looks good.  it isn't necessary to have the if/else on either question.  you can just write `let numHolder = arguments[0];`